manual/manual/tutorials/objectexamples.etex

```diff
@@ -2,15 +2,20 @@
 \label{c:objectexamples}
 \pdfchapterfold{-15}{Tutorial: Objects in OCaml}
 %HEVEA\cutname{objectexamples.html}
-{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
+{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
 
 \bigskip
 
 \noindent This chapter gives an overview of the object-oriented features of
-OCaml. Note that the relation between object, class and type
-in OCaml is very different from that in mainstream
-object-oriented languages like Java or C++, so that you should not
-assume that similar keywords mean the same thing.
+OCaml.
+
+Note that the relationship between object, class and type in OCaml is different than
+in Java and C++, so you shouldn't assume that similar keywords mean the same thing.
+Object-oriented features are used much less frequently in OCaml than
+in mainstream object-oriented languages such as Java and C++.  OCaml has other
+alternatives that are often more appropriate, such as modules and functors.  Many
+OCaml programs do not use objects at all.
```

Comment:
Perhaps "Indeed, many OCaml programs do not use the object system at all."?

Reply:
I added "indeed" but kept "objects".